scripts/deblur

```diff
@@ -386,57 +432,85 @@ def build_biom_table(seqs_fp, output_biom_fp, file_type, log_level, log_file):
 @click.option('--delim', required=False, type=str, default='_',
               show_default=True, help="Delimiter in FASTA labels to separate "
                                       "sample ID from sequence ID")
-@click.option('--keep-tmp-files', required=False, type=bool, default=False, is_flag=True,
+@click.option('--keep-tmp-files', required=False, type=bool, default=False,
+              is_flag=True,
               show_default=True,
               help="Keep temporary files (useful for debugging)")
-@click.option('--log-level', required=False, type=click.IntRange(1, 5, clamp=True), default=2,
+@click.option('--log-level', required=False,
+              type=click.IntRange(1, 5, clamp=True), default=2,
               show_default=True, help="Level of messages for log file"
               "(range 1-debug to 5-critical")
-@click.option('--log-file', required=False, type=click.Path(resolve_path=True, readable=True, exists=False,
-              dir_okay=True), default='deblur.log',
+@click.option('--log-file', required=False,
+              type=click.Path(resolve_path=True, readable=True,
+                              exists=False, dir_okay=True),
+              default='deblur.log',
               show_default=True, help="log file name")
+@click.option('--jobs-to-start', '-O', required=False, type=int, default=1,
+              show_default=True,
+              help="Number of jobs to start (if to run in parallel)")
+@click.option('--is-worker-thread', required=False, type=bool, default=False,
```

Comment:
I think this option can mislead users. Usually, in multiprocessed programs, each thread has its own id (see [this](http://stackoverflow.com/a/10192611/3746629) SO answer to see how to get it in python multiprocessing) and then the thread with lowest ID (in this case 1) is the master thread, while the other ones are working threads. You can use that to remove this option and just manage that internally.